includes/Core/Modules/REST_Modules_Controller.php

```diff
@@ -0,0 +1,756 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Modules\REST_Modules_Controller
+ *
+ * @package   Google\Site_Kit\Core\Modules
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Modules;
+
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\REST_API\REST_Routes;
+use Google\Site_Kit\Core\REST_API\REST_Route;
+use Google\Site_Kit\Core\REST_API\Exception\Invalid_Datapoint_Exception;
+use WP_REST_Server;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_Error;
+use Exception;
+
+/**
+ * Class for handling modules rest routes.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class REST_Modules_Controller {
+
+	const REST_ENDPOINT_CHECK_ACCESS = 'core/modules/data/check-access';
```

Comment:
I realize this was how it was before, but this is technically the "route"; an endpoint is more specific to include the method. This also corresponds to the `$route` argument of [`register_rest_route`](https://developer.wordpress.org/reference/functions/register_rest_route/).

Reply:
Updated to say `REST_ROUTE_CHECK_ACCESS`, thanks!